shop/src/util/sort.js

```diff
@@ -0,0 +1,16 @@
+export const sort = ({ optionChosen, disItemAfterChosen }) => {
+    switch (optionChosen) {
+      case "retail_price_cents":
+          disItemAfterChosen = disItemAfterChosen.sort((a, b) => (a.retail_price_cents > b.retail_price_cents ? 1 : -1));
+          break;
```

Comment:
are we not returning anything in this case?

Reply:
We return disItemAfterChosen. Default value of disItemAfterChosen is allShoesItems. After sort, I put data which already sort in disItemAfterChosen also.